i6_models/assemblies/util.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import MISSING, dataclass
```

Comment:
This should not live in assemblies. It can also be used in parts. I would put it in /i6_models/config.py